sql/moz-fx-data-shared-prod/firefox_ios_derived/new_profile_activation_v1/query.sql

```diff
@@ -39,7 +39,7 @@ client_search AS (
     AND normalized_app_name = 'Fennec'
```

Comment:
Wow, we really have to change this string for `normalized_app_name`

Reply:
I'm not sure if filtering just for iOS is enough here or if we also need to filter for a specific app_name here?

Reply:
Given that eventually we have a join on client_id and those are "probably" unique it shouldn't give us wrong numbers, but we do have other iOS apps on the table (Klar, Focus, etc) so if we don't filter it's technically a less efficient join later.

Reply:
So what is still unclear to me, should we be filtering for a specific app here or are we doing a bag consisting of everything ios?

Reply:
Just a single app. This should only be Firefox for iOS, not Focus for iOS.

Reply:
ok, I added back the filter for Fennec